### DISCUSSION TITLE
[Feature] Consider adding mariadb/mysql support

### DISCUSSION THREAD

**Discussion by dada051** [immich-app/immich#3581] [Feature Request]
### The feature

I think that having more choices in the database backend will improve user base, and MariaDB is a great option for that. I personally run a lot of containerized applications and lot of them use MariaDB (the others that use DB use SQLite), none of them use Postgres. I understand that implementing SQLite can be a lot of work, but I hope that it's not the case for MariaDB.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3434 / #562

**Reply by dada051** [reply]
Sorry I searched for mariadb but probably not in the right place.
